rules/Php55/Rector/String_/StringClassNameToClassConstantRector.php

```diff
@@ -139,17 +144,12 @@ public function provideMinPhpVersion(): int
         return PhpVersionFeature::CLASSNAME_CONSTANT;
     }
 
-    private function shouldSkip(string $classLikeName, String_ $string): bool
+    private function shouldSkip(string $classLikeName): bool
     {
         if (! $this->reflectionProvider->hasClass($classLikeName)) {
```

Comment:
looks like this check can be dropped now

Reply:
Good catch :+1: